assets/src/web-stories-block/css/core-themes.css

```diff
@@ -0,0 +1,9 @@
+@import 'core-themes/twenty-twenty-one.css';
+@import 'core-themes/twenty-twenty.css';
+@import 'core-themes/twenty-seventeen.css';
+@import 'core-themes/twenty-sixteen.css';
+@import 'core-themes/twenty-fifteen.css';
+@import 'core-themes/twenty-fourteen.css';
+@import 'core-themes/twenty-twelve.css';
+@import 'core-themes/twenty-eleven.css';
+@import 'core-themes/twenty-ten.css';
```

Comment:
I personally don't like this. It means lots of CSS is loaded when it not needed. Can you only load this CSS file is say is `twenty-ten` is active. Should be done in PHP.

Reply:
Removed single stylesheet in favour of theme specific ones.